packages/react-native-editor/__device-tests__/helpers/device-config.json

```diff
@@ -1,16 +1,14 @@
 {
 	"ios": {
 		"local": {
-			"deviceName": "iPhone 14",
+			"deviceName": "iPhone 15",
 			"deviceTabletName": "iPad (10th generation)",
-			"platformVersion": "16.2"
+			"platformVersion": "17.2"
 		},
 		"saucelabs": {
-			"deviceName": "iPhone 14 Simulator",
-			"deviceTabletName": "iPad (10th generation) Simulator"
-		},
-		"buildkite": {
-			"platformVersion": "16.4"
+			"deviceName": "iPhone 15 Simulator",
+			"deviceTabletName": "iPad (10th generation) Simulator",
+			"platformVersion": "17.0"
```

Comment:
For reference, these parameters are no longer used in Gutenberg Mobile after [this change](https://github.com/wordpress-mobile/gutenberg-mobile/pull/6525/commits/51d6ef1f1b2382168a9c136c9b4534635ad50cbb#diff-f5d79364f817ebab2040ffe8bb5bcec94793a7206ae45ab6a903e6749a93d4d4R16).

Reply:
Ideally, we should use the same parameters in both local and SauceLabs runs. However, I'm not sure if this will be always the case, as it will depend on the devices supported in SauceLabs. That said, we might consider reverting [this change](https://github.com/wordpress-mobile/gutenberg-mobile/pull/6525/commits/51d6ef1f1b2382168a9c136c9b4534635ad50cbb#diff-f5d79364f817ebab2040ffe8bb5bcec94793a7206ae45ab6a903e6749a93d4d4) to be able to run E2E tests locally and in CI.

Reply:
Yeah, I've removed the `buildkite` config and moved the `platformVersion` to the SauceLabs one since they [don't support](https://saucelabs.com/products/platform-configurator#/) `17.2` simulators yet. Ideally, we will have the same version once we run the tests in our CI environment.

>we might consider reverting this change to be able to run E2E tests locally and in CI.

Yup something to consider for sure!